examples/lambda-pets/src/infrastructure/petfinder.ts

```diff
@@ -0,0 +1,51 @@
+import { doFx, Fx, get } from '../../../../src'
+import { GeoLocation, GetPets, Pets } from '../domain/model'
+import { getJson, HttpEnv, postJson } from './http'
+
+// Petfinder types and APIs
+// Note that getPets is a concrete implementation of
+// the GetPets domain interface and instantiates the specific
+// set of effects that it requires.
+
+export type PetfinderPets = {
+  animals: readonly Animal[]
+}
+
+export type Animal = {
```

Comment:
What are these types for and why are they defined here?

Reply:
They define the type of data returned by the Petfinder API.  So, they're Petfinder-specific, and really only used in this file.